frontend/storage/models/Image.ts

```diff
@@ -1,4 +1,5 @@
 export type ImageProps = Required<Image>;
+import UndefinedCover from '../../assets/cover_undefined.jpg';
```

Comment:
Go with `UndefinedCoverPath` (since it actually is a path)

Edit: Actually go with `DefaultImagePath`, more below.